api/v1alpha1/selfnoderemediationconfig_webhook_test.go

```diff
@@ -71,6 +73,41 @@ var _ = Describe("SelfNodeRemediationConfig Validation", func() {
 		// test update validation on a valid CR
 		testValidCR("update")
 
+		Context("SafeTimeToAssumeNodeRebootedSeconds Validation", func() {
+			var oldSnrc, newSnrc *SelfNodeRemediationConfig
```

Comment:
Not sure it'll work well for this case, I was thinking of two reasons:
-  I think the variable name should have indication of whether the configuration is before (old) or after (new) the user change since we completely ignore the old one, adding that will make a pretty long var name.
- I'm using the same variable for several tests  (in one of them the updated configuration is without annotation) so embedding the annotation in the variable name means creating multiple variables - not terrible but not ideal IMO.